### ISSUE TITLE
windows-x64 and node-webkit >0.14.6

### ISSUE THREAD

**Issue by wehowski** [nwjs/nw.js#4976]
Hello everybody,
we have the following issue:
https://github.com/jyapayne/Web2Executable/issues/142

I use this compiler web2exe and the win64 compilation does not work if I compile nw.js >0.14.6
win32 works, the other systems I did not test.

**Comment by ghostoy** [comment]
@wehowski Did you turn on the compression of Web2Exe? The compression is based on [UPX](http://upx.sourceforge.net/) and currently it doesn't work with latest NW.js. You can temporarily turn off the compression to workaround.

I'm opening an issue on jyapayne/Web2Executable#153 to suggest compressing .dll files instead of nw.exe to achieve better compression ratio without crash.

**Comment by wehowski** [comment]
Yes, thats it.
To check the option "uncompressed folder" did not help, but setting compression level to 0 made it work!
Thank you!

**Comment by ghostoy** [comment]
@wehowski Yes, I mean the compression level. Not "uncompressed folder". Sorry for the misunderstanding.
